opencog/atomspace/AtomSpace.cc

```diff
@@ -99,6 +99,156 @@ AtomSpace& AtomSpace::operator=(const AtomSpace&)
          "AtomSpace - Cannot copy an object of this class");
 }
 
+bool AtomSpace::compare_atomspaces(const AtomSpace& space_first,
+                                   const AtomSpace& space_second,
+                                   bool check_truth_values,
+                                   bool emit_diagnostics)
+{
```

Comment:
Moving this somewhere else makes sense.

We don't really have a test-specfic utilities directory. Should I create called `testutils`? 

Or would you prefer that I just create a file TestUtilities.h and place that in `atomspaceutils` like I did with `RandomAtomGenerator`?